### PR TITLE
Improve 3DS UX/UI

### PR DESCRIPTION
**Pull request by EpokK**

### CODE REVIEW

File: containers/payments/PaymentVerificationModal.js [hunk 1]
```diff
@@ -30,19 +68,66 @@ const PaymentVerificationModal = ({ params, token, approvalURL, onSubmit, ...res
...
             {...rest}
         >
-            {loading ? <Loader /> : <img src={tabSvg} alt={c('Title').t`New tab`} />}
-            {loading ? (
-                <Alert>{c('Info').t`Please verify the payment in the new tab.`}</Alert>
-            ) : (
-                <Alert>{c('Info')
-                    .t`A new tab will open to confirm the payment, please disable any popup blockers.`}</Alert>
-            )}
+            {step === STEPS.REDIRECT ? (
```
Comment (mmso) [inline containers/payments/PaymentVerificationModal.js]:
```suggestion
            {{
                [STEPS.REDIRECT]: (
                    <>
                        <p className="aligncenter">{c('Info')
                            .t`Your bank requires 3-D Secure verification for security purposes.`}</p>
                        <p className="aligncenter">
                            <img src={tabSvg} alt={c('Title').t`New tab`} />
                        </p>
                        <Alert>{c('Info')
                            .t`Verification will open a new tab. Please disable any popup blockers. <b>You will not be charged</b>. Any amount used to verify your card will be refunded immediately.`}</Alert>
                    </>
                ),
                [STEPS.REDIRECTING]: (
                    <>
                        <p className="aligncenter">{c('Info')
                            .t`You will be soon redirected to your bank to verify your payment.`}</p>
                        <Loader />
                        <Alert>{c('Info').t`Verification will open a new tab, please disable any popup blockers.`}</Alert>
                    </>
                )
                ,
                ...
            }[step]}
```

File: containers/payments/PaymentVerificationModal.js [hunk 2]
```diff
@@ -1,26 +1,64 @@
...
+    REDIRECTED: 'redirected',
+    FAIL: 'fail'
+};
+
+const PROCESSING_DELAY = 5000;
+
 const PaymentVerificationModal = ({ params, token, approvalURL, onSubmit, ...rest }) => {
-    const [loading, withLoading] = useLoading();
+    const TITLES = {
+        redirect: c('Title').t`Card verification`,
```
Comment (mmso) [inline containers/payments/PaymentVerificationModal.js]:
```suggestion
        [STEPS.REDIRECT]: c('Title').t`Card verification`,
        ...
```

File: containers/payments/PaymentVerificationModal.js [hunk 3]
```diff
@@ -1,26 +1,64 @@
...
+        rest.onClose();
+    };
 
     const handleSubmit = async () => {
         try {
-            await process({ Token: token, api, approvalURL, secureURL });
+            setStep(STEPS.REDIRECTING);
+            setTimeout(() => {
+                setStep(STEPS.REDIRECTED);
+            }, PROCESSING_DELAY);
```
Comment (vincaslt) [inline containers/payments/PaymentVerificationModal.js]: Why do we wait 5 seconds instead of for some successful response?

Comment (jovanjovanovski) [reply containers/payments/PaymentVerificationModal.js]: The problem is if there is no successful response (eg. pop-up blocker).

**Pull request merged**